promql/functions.go

```diff
@@ -298,6 +299,14 @@ func funcSortDesc(ev *evaluator, args Expressions) model.Value {
 	return vector(byValueSorter)
 }
 
+// === sort_by_label(node model.ValVector, model.ValString) model.ValVector ===
+func funcSortByLabel(ev *evaluator, args Expressions) model.Value {
+	l := ev.evalString(args[1])
+	s := vectorByLabelHeap{vector: ev.evalVector(args[0]), labelName: l.String()}
+	sort.Sort(s)
```

Comment:
Unlike values, a timeseries can have multiple labels. If I want to sort by A, then B I can do `sort_by_label(sort_by_label(my_vector, "B") "A")` but only if the sort is stable.